libs/settings/api/src/lib/settings.resolver.ts

```diff
@@ -19,13 +19,13 @@ export class SettingsResolver {
   }
 
   @Query(returns => Setting, {
-    name: 'setting',
+    name: 'settingById',
```

Comment:
Can we revert this? As this is iirc inconsistent with the rest of the application.

Reply:
This is actually a conflict in graphql schema. v1 searches by name, v2 by id.